capstone/src/main/webapp/index.html

```diff
@@ -17,6 +17,17 @@ <h1 id="explore-heading">Explore Clubs</h1>
           <option value="alpha">Alphabetical</option>
           <option value="size">Size</option>
         </select>
+        
```

Comment:
nit: Seems like there are a few empty lines here. If they are not needed, please remove them!